### PR TITLE
chore: suppress dotnet logo

### PR DESCRIPTION
**Pull request by BusHero**
Set the `DOTNET_NOLOGO` environment variable to 1 to suppress annoying dotnet logo message

I confirm that the pull-request:

- [X] Follows the contribution guidelines
- [X] Is based on my own work
- [X] Is in compliance with my employer

### CODE REVIEW

File: source/Nuke.GlobalTool/templates/build.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ DOTNET_CHANNEL="STS"
 export DOTNET_CLI_TELEMETRY_OPTOUT=1
 export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 export DOTNET_MULTILEVEL_LOOKUP=0
-
+export DOTNET_NOLOGO=1
```
Comment (TheBlueSky) [inline source/Nuke.GlobalTool/templates/build.sh:20]:
```suggestion
export DOTNET_NOLOGO=1

```

**Comment by matkoch** [comment]
Which output does this concern exactly?

**Comment by TheBlueSky** [comment]
> Which output does this concern exactly?

As per .NET SDK documentation (see [here](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-environment-variables#dotnet_skip_first_time_experience)), `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` is unsupported since .NET Core 3.0 and was replaced by `DOTNET_NOLOGO`.

**Comment by matkoch** [comment]
That wasn't my question. I was asking what output is shown if this is not set.

**Comment by TheBlueSky** [comment]
The same output that not setting `DOTNET_SKIP_FIRST_TIME_EXPERIENCE` prior to .NET Core 3.0, which is the welcome message about the telemetry and some information and links.

**Comment by matkoch** [comment]
You're not answering my question, and I'm not trying to be difficult either. I want to see the exact output that is suppressed by this variable to make sure that the SDK version is still included in the output as I find this is relevant for builds.

**Comment by TheBlueSky** [comment]
Something like this:

```
Welcome to .NET 8.0!
---------------------
SDK Version: 8.0.100

Telemetry
---------
The .NET tools collect usage data in order to help us improve your experience. It is collected by Microsoft and shared with the community. You can opt-out of telemetry by setting the DOTNET_CLI_TELEMETRY_OPTOUT environment variable to '1' or 'true' using your favorite shell.

Read more about .NET CLI Tools telemetry: https://aka.ms/dotnet-cli-telemetry

----------------
Installed an ASP.NET Core HTTPS development certificate.
To trust the certificate, run 'dotnet dev-certs https --trust'
Learn about HTTPS: https://aka.ms/dotnet-https

----------------
Write your first app: https://aka.ms/dotnet-hello-world
Find out what's new: https://aka.ms/dotnet-whats-new
Explore documentation: https://aka.ms/dotnet-docs
Report issues and find source on GitHub: https://github.com/dotnet/core
Use 'dotnet --help' to see available commands or visit: https://aka.ms/dotnet-cli
--------------------------------------------------------------------------------------
```

**Comment by matkoch** [comment]
Thanks

**Pull request merged**